docs/streams/developer-guide/dsl-api.html

```diff
@@ -870,6 +870,12 @@ <h4 class="anchor-heading"><a id="streams_concepts_globalktable" class="anchor-l
                 <p id="streams-developer-guide-dsl-transformations-stateful-overview">Stateful transformations depend on state for processing inputs and producing outputs and require a <a class="reference internal" href="../architecture.html#streams_architecture_state"><span class="std std-ref">state store</span></a> associated with the stream processor. For example, in aggregating operations, a windowing state store is used to collect the latest aggregation results per
                     window. In join operations, a windowing state store is used to collect all of the records received so far within the
                     defined window boundary.</p>
+		<p><b>Note:</b> Following store types are used regardless of the possibly specified type (via the parameter <code class="docutils literal"><span class="pre">materialized</span></code>):
+		<ul class="simple">
+		    <li>non-windowed aggregations and plain KTables use <a class="reference external" href="/{{version}}/javadoc/org/apache/kafka/streams/state/TimestampedKeyValueStore.html">TimestampedKeyValueStore</a>s</li>
```

Comment:
Maybe we should say `non-windowed KTable` -- "plain" does not sound technical.

Reply:
OK, will do.